storage-proofs/src/zigzag_graph.rs

```diff
@@ -175,6 +175,9 @@ impl<Z: ZigZag> Graph<Z::BaseHasher> for Z {
         }
         assert!(parents.len() == self.degree());
         parents.sort();
+        if self.reversed() {
+            parents.reverse();
+        }
```

Comment:
For efficiency, we should probably sort either ascending or descending depending on layer direction rather than sorting then reversing.